include/ultrasonic.h

```diff
@@ -4,7 +4,7 @@
 #include <stdint.h>
 
 void setup_ultrasonic(void);
-
+bool is_ultrasonic_echo_high(void);
```

Comment:
Once again, justify why you need this as existing specification should be sufficient.
I'm happy to make changes but I don't see why this change is needed.

Reply:
I will remove it as you won't use it in main.cpp. That was my bad.